### PR TITLE
Mpuncel/label 404 error

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

File: pkg/labels/consul_applicator_test.go [hunk 1]
```diff
@@ -331,3 +338,57 @@ func TestWatchMatchDiff(t *testing.T) {
...
+	if rcLabels[0].LabelType != RC {
+		t.Errorf("returned label results should have had type %q but was %q", RC, rcLabels[0].LabelType)
+	}
+
+	if rcLabels[0].Labels["some_key"] != "some_value" {
+		t.Errorf("expected label result to have value %q for key %q but was %q", "some_value", "some_key", rcLabels[0].Labels["some_key"])
+	}
+}
+
+func XTestListLabels404(t *testing.T) {
```
Comment (petertseng) [inline pkg/labels/consul_applicator_test.go]: x!

File: pkg/labels/consul_applicator_test.go [hunk 2]
```diff
@@ -331,3 +341,57 @@ func TestWatchMatchDiff(t *testing.T) {
...
```
Comment (rudle) [inline pkg/labels/consul_applicator_test.go]: :eyes:

Comment (rudle) [inline pkg/labels/consul_applicator_test.go]: Good news is that it passes.

**Comment by mpuncel** [comment]
fixed the test name

**Pull request merged**